source/load.py

```diff
@@ -0,0 +1,50 @@
+from transformers import GPT2LMHeadModel
+from transformers import PreTrainedTokenizerFast
+import os
+
+
+class LoadModelFromLocalFolder:
```

Comment:
referring to comment before, maybe one class to rule them all? and a logic to choose local or cloud? makes sense oder nicht?

Reply:
yeah yeah yeah